c/src/ml-api-service-offloading.c

```diff
@@ -25,9 +25,22 @@
 #include "ml-api-service.h"
 #include "ml-api-service-private.h"
 #include "ml-api-service-offloading.h"
+#include "ml-api-service-training-offloading.h"
 
 #define MAX_PORT_NUM_LEN 6U
 
+/**
+ * @brief Enumeration for ml-offloading mode type.
+ */
+typedef enum
+{
+  ML_OFFLOADING_MODE_TYPE_DEFAULT = 0,
+  ML_OFFLOADING_MODE_TYPE_TRAINING = 1,
+
+  ML_OFFLOADING_TYPE_MAX
+} ml_offloading_mode_type_e;
```

Comment:
ml_service_offloading_mode_type_e for consistency?

Reply:
Thank you. changed.